apps/admin-x-activitypub/src/views/Notifications/Notifications.tsx

```
@@ -303,36 +296,49 @@ const Notifications: React.FC = () => {
                                                 {(
                                                     (group.type === 'reply' && group.inReplyTo) ||
                                                     (group.type === 'like' && !group.post?.name && group.post?.content) ||
                                                     (group.type === 'repost' && !group.post?.name && group.post?.content)
                                                 ) && (
-                                                    <div
-                                                        dangerouslySetInnerHTML={{__html: stripHtml(group.post?.content || '')}}
-                                                        className='ap-note-content mt-1 line-clamp-2 text-pretty text-gray-700 dark:text-gray-600'
-                                                    />
+                                                    (group.post?.type === 'note' ?
+                                                        <div
+                                                            dangerouslySetInnerHTML={{__html: stripHtml(group.post?.content || '')}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)